client/trino-cli/src/main/java/io/trino/cli/ClientOptions.java

```diff
@@ -121,6 +121,9 @@
     @Option(names = "--client-tags", paramLabel = "<tags>", description = "Client tags")
     public String clientTags;
 
+    @Option(names = "--extra-headers", paramLabel = "<header=value>", description = "Extra HTTP headers to attach to Trino queries")
```

Comment:
Can you make the label and description consistent with other options that have composite values? Like `--resource-estimate` or `--extra-credential`.

Reply:
Will do!

Reply:
Updated; I've made it consistent with the `--client-tags` flag above as it's a close cousin.